io.liri.Calculator.yaml

```diff
@@ -0,0 +1,34 @@
+app-id: io.liri.Calculator
+branch: stable
+runtime: org.kde.Platform
+runtime-version: "5.11"
+sdk: org.kde.Sdk
+base: io.liri.BaseApp
+base-version: "5.11"
+command: liri-calculator
+finish-args:
+- --device=dri
+- --socket=wayland
+- --socket=x11
+- --share=ipc
+- --env=SESSION_MANAGER=
+cleanup:
```

Comment:
I presume none of these are needed.

Reply:
`SESSION_MANAGER` was removed to silence this Qt error message:
`Qt: Session management error: Could not open network socket`

The `--socket=x11` and `--share=ipc` are need to run the app on X11.

Reply:
I was referring to the `cleanup` list in all of these apps.

Reply:
Done

Reply:
The message is harmless though so I don't see much gain in silencing it.

Reply:
Done